bin/i18n/resources/dashboard/course_offerings/sync_out.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require 'json'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_out_base'
+require_relative '../course_offerings'
+
+module I18n
+  module Resources
+    module Dashboard
+      module CourseOfferings
+        class SyncOut < I18n::Utils::SyncOutBase
+          process do |language|
+            crowdin_file_path = I18nScriptUtils.locale_dir(language[:crowdin_name_s], DIR_NAME, FILE_NAME)
+            next unless File.exist?(crowdin_file_path)
+
+            i18n_locale = language[:locale_s]
+            distribute_localization(i18n_locale, crowdin_file_path) unless I18nScriptUtils.source_lang?(language)
+
+            i18n_file_path = I18nScriptUtils.locale_dir(i18n_locale, DIR_NAME, FILE_NAME)
+            I18nScriptUtils.move_file(crowdin_file_path, i18n_file_path)
```

Comment:
On the first read, I read I was confused here. We have a `distribute_localization` call, but then later we make a call to `I18nScriptUtils.move_file`. I was confused why we are moving things twice.

Reply:
After reading it some more, is the final move to `i18n/locales` kind of how we mark the file as processed?

Reply:
Yes, before the modularizing, we copied files from the `i18n/locales/Spanish` dir (Crowdin locale dir) to the `i18n/locales/es-ES` dir (I18n locale dir) first then check if a file was "changed" and only then distribute it.

But since we download only "changed" files from Crowdin now, we distribute files directly from the Crowdin locale dir and only then move them to I18n locale dir to kind of "mark them as processed".

Do you know why we move and commit files in the I18n locale dir at the first place?

Reply:
I don't know the original intent. I'm guessing it was a simple way to track what has been processed.